holoviews/core/operation.py

```diff
@@ -70,28 +70,45 @@ def get_overlay_bounds(cls, overlay):
             raise ValueError("Extents across the overlay are inconsistent")
 
 
-class DynamicOperation(Operation):
+class Dynamic(Operation):
```

Comment:
Should probably be a ``ParameterizedFunction`` and moved to some other file as a utility..